app/src/main/java/com/github/featuredetectandroid/ui/Menu.kt

```diff
@@ -0,0 +1,61 @@
+package com.github.featuredetectandroid.ui
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.selection.selectable
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.RadioButton
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Alignment.Companion.CenterVertically
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.github.featuredetectandroid.utils.KeypointDetectionAlgorithm
+
+@Composable
+fun Menu(selectedAlgorithm: String, onAlgorithmSelected: (String) -> Unit) {
+    val radioOptions = KeypointDetectionAlgorithm.names
+
+    Row(
+        horizontalArrangement = Arrangement.Center,
+        verticalAlignment = Alignment.Bottom
+    ) {
+        Text(
+            text = "Keypoint detection algorithm:",
+            modifier = Modifier.padding(all = 20.dp),
+            textAlign = TextAlign.Start,
+            style = MaterialTheme.typography.h6
+        )
+    }
+
+    radioOptions.forEach { text ->
```

Comment:
Let's use something more representative here instead of `text`. I'm not sure whta, though, as for the code below `algorithm` would suite best but then it's better to rename `radioOptions` to `algorithmNames` and then what's the point of creating this value if we can just inline `KeypointDetectionAlgorithm.names` as it has a similar meaning. The other option is to raname `text` to `option` but then it looks not so good in the code below.

To sum up, I'd go with `option` if you want `radioOptions` to stay.

Reply:
Or we can just pass the header text and radio options as parameters and rename everything to more general words so that this menu doesn't assume that it's used for selecting an algorithm

Reply:
> Or we can just pass the header text and radio options as parameters and rename everything to more general words so that this menu doesn't assume that it's used for selecting an algorithm.
> 
> If you do so, I'd suggest the following argument naming and order:
> 
> * `"Keypoint detection algorithm:"` -> `header`
> * `radioOptions` -> `options`
> * `selectedAlgorithm` -> `selectedOption`
> * `onAlgorithmSelected` -> `onSelected`

Good idea! Implemented.